SPECS/varstored.spec

```diff
@@ -64,17 +59,25 @@ when the guest is not running.
 %autosetup -p1
 %{?_cov_prepare}
 
-# Check for pem files in the source archive
-if find certs -name "*.pem" | grep -q pem; then
-  echo "pem files are present in the source archive"
-  echo "Please remove them using the script SOURCES/remove-certs-from-tarball.sh"
-  echo "and push the updated source archive"
-  false
-fi
 
 %build
+# XCP-ng: inject our certs/ directory and cert list
+rm -rf certs
+cp -r %{_sourcedir}/certs -t .
+echo \
+     \"certs/KEK/Certificates/MicCorKEKCA2011_2011-06-24.der\" \
+     \"certs/KEK/Certificates/microsoft corporation kek 2k ca 2023.der\" \
+     > KEK.list
+echo \
+     \"certs/DB/Certificates/MicWinProPCA2011_2011-10-19.der\" \
+     \"certs/DB/Certificates/windows uefi ca 2023.der\" \
+     \"certs/DB/Certificates/MicCorUEFCA2011_2011-06-27.der\" \
+     \"certs/DB/Certificates/microsoft uefi ca 2023.der\" \
+     \"certs/DB/Certificates/microsoft option rom uefi ca 2023.der\" \
+     > db.list
```

Comment:
Accessing `%{_sourcedir}` breaks RPM packaging rules. We need to add a Source1, Source2, etc, or another way to that doesn't directly access `%{_sourcedir}` outside any declaration of the source files in the spec file.

Also, modifying the certs directory, the KEK list and the db list likely belongs to the `%prep` section.

Reply:
How do I specify `--cert-search-path` later in the `gen-dbx.py` step, maybe with `dirname`?

Reply:
Moved everything to `%{SOURCE#}`.